doc/classes/PopupMenu.xml

```diff
@@ -186,7 +186,7 @@
 			<param index="1" name="submenu" type="String" />
 			<param index="2" name="id" type="int" default="-1" />
 			<description>
-				Adds an item that will act as a submenu of the parent [PopupMenu] node when clicked. The [param submenu] argument is the name of the child [PopupMenu] node that will be shown when the item is clicked.
+				Adds an item that will act as a submenu of the parent [PopupMenu] node when clicked. The [param submenu] argument should be the name of an existing [PopupMenu] that has been previously created and added as a child to the parent [PopupMenu] node, it will be shown when the item is clicked.
```

Comment:
If this is a hard requirement, I suggest using "must":

```suggestion
				Adds an item that will act as a submenu of the parent [PopupMenu] node when clicked. The [param submenu] argument must be the name of an existing [PopupMenu] that has been previously created and added as a child to the parent [PopupMenu] node, it will be shown when the item is clicked.
```

Reply:
It is, it's where the confusion comes from. Thank for your suggestion.